src/main/java/htsjdk/tribble/util/URLHelper.java

```diff
@@ -33,13 +35,27 @@
  */
 public interface URLHelper {
 
+    /**
+     * @return URL of the associated resource
+     */
     URL getUrl();
 
+    /**
+     * @return content length of the resource, or -1 if not available
+     * @throws IOException
+     */
     long getContentLength() throws IOException;
 
+    /**
+     * Open an InputStream to stream the contents of the resource
+     * @return
+     * @throws IOException
+     */
     InputStream openInputStream() throws IOException;
 
     /**
+     * Open an InputStream to stream a slice (range)  of the resource.
+     *
      * May throw an OperationUnsupportedException
      * @deprecated Will be removed in a future release, as is somewhat fragile
```

Comment:
This line saying it's deprecated should be removed too.

Reply:
removed